docs/contributing.md

```diff
@@ -0,0 +1,78 @@
+# Contributing
+
+We love contributions from the community! This document explains how you can contribute to the project.
```

Comment:
Not against community contributions, but we're not well established to support this right now - it would probably be a good task to consider how an external party could maintain their own server, produce better docs to support this and then write a contribution guide. Could we rephrase this file (and maybe combine for now with installation) to be a "how to setup dev environment for metamist"?